src/client/client.h

```diff
@@ -349,6 +349,7 @@ class Client : public con::PeerHandler, public InventoryManager, public IGameDef
 	void confirmRegistration();
 	bool m_is_registration_confirmation_state = false;
 	bool m_simple_singleplayer_mode;
+	bool m_enable_register_confirmation;
```

Comment:
no need to cache this setting it's not called everytime.

Reply:
AFK right now. Is `g_settings` available in this scope?

Reply:
Done.